### ISSUE TITLE
Print lanyard cards

### ISSUE THREAD

**Issue by dbrgn** [SwissPy/swisspycon#54]
Cards where people can write their name on.

**Comment by ccdesales** [comment]
This is how the lanyards look like:

![screen shot 2016-01-06 at 14 03 07](https://cloud.githubusercontent.com/assets/1451785/12143124/4a6af52c-b47e-11e5-94fe-b179a2ab13b8.png)

**Comment by dbrgn** [comment]
Could someone pick up this task and design simple cards with our logo and a field for the name that we can print?

**Comment by jakeret** [comment]
I think it's sufficient to rescale our poster and replace the text with a name box.. 
(If we print them we also need to cut them in shape and stich a hole)

**Comment by dbrgn** [comment]
We can print and cut them at HSR, they have tools for that.

**Comment by href** [comment]
I can do it. What's the size of the card? Also if someone could mail me all the names for the cards I'll create a PDF ready to print with all names.

**Comment by dbrgn** [comment]
I wouldn't pre-fill the names. If someone cancels or someone new attends, then we'd have to throw some of them away.

I'd create two or three empty fields though: Name, Company and maybe Twitter or Github handle.

Regarding the sleeves, @ccdesales, did you take a look at a shop like officeworld? What sizes can we get?

**Comment by cdesales** [comment]
No, I haven't checked since it was not until recently that we knew exactly the kind of lanyards we'd get.

**Comment by jakeret** [comment]
> What's the size of the card?

Here are the dimensions (W x H) of cards I've got:
- Europython: 6.8x8.5cm
- SciPy:  11x9.5cm

I liked the EuroPython card

**Comment by dbrgn** [comment]
Well, could one of you take a look at concrete sleeves from an office shop with reasonable pricing (and self-assign this ticket)? Once we have those we can create the design :)

**Comment by href** [comment]
I think company name and name is enough by the way. Better to have some space for the information so people can write in large letters.

**Comment by href** [comment]
Those look okay: http://shop.manroof.ch/collections/ausweishullen/products/ausweishullen-backstage-format (the PP variant).

I think it would be best to order them and send them to you @dbrgn. Since it sounds like you're going to be handling the printing and cutting. I'm guessing 130 pieces is enough?

If we all agree I can send out the order to your address if you can mail it to me.

**Comment by jakeret** [comment]
> I think company name and name is enough by the way. Better to have some space for the information so people can write in large letters.

agree

Looks good. Alternatively: http://www.amazon.de/Ausweish%C3%BCllen-vertikal-aus-Vinyl-Formate/dp/B01129UWX2/ref=pd_sim_sbs_229_2?ie=UTF8&dpID=312BFOl8N-L&dpSrc=sims&preST=_AC_UL160_SR160%2C160_&refRID=1E3JNVF7CRRJFB61V9YE

We could assemble them at the pre-conference meeting. Shouldn't take too much time if we are 5 or 6 persons

**Comment by dbrgn** [comment]
> the PP variant

What do you mean with "PP"?

Yeah, they look fine to me. The only problem _might_ be that the hole is long and not circular. But I guess that should work fine.

I'll send you my mailing address.

**Comment by href** [comment]
> What do you mean with "PP"?

If you have a look at the variants you see that they have PVC and PP. Those are two different kinds of plastic.

> The only problem might be that the hole is long and not circular. 

I think most are like that.

I'm inclined to order the Amazon ones at this point because they're cheaper. I just hope they make it in time.

If not we can still order another batch somewhere else if Amazon is underestimating the time it takes to send those to us and we are without sleeves 5 days before the conference. Though their estimation should be good.

By the way, how about pre-printing all the names anyway and laminating all the passes? Assuming that HSR has a machine to do that.

**Comment by jakeret** [comment]
> I just hope they make it in time.

I also thought about that. But in principle it should be ok

> By the way, how about pre-printing all the names anyway and laminating all the passes? Assuming that HSR has a machine to do that.

It would at least look more professional than hand written names. But at the same time it makes the registration process more complex

**Comment by href** [comment]
Actually since we need 3 packages we end up paying import taxes so it's not really cheaper :)

@dbrgn Go with my original proposal or laminate?

**Comment by href** [comment]
I did a quick design for the passes. I think this should suffice. I didn't include any logos because they will not look too great at that size, there's already a logo on the lanyard and all sponsors are getting something out of it already:

![screenshot of sketch 20-01-16 14-42-30](https://cloud.githubusercontent.com/assets/273163/12450481/4553c7ce-bf84-11e5-8268-2ef33d4603b1.png)

**Comment by jakeret** [comment]
Geeky, like it! ;-) I think it would be good to have the date and location on the card as well. Maybe somewhere in the blue area. 

> Go with my original proposal

I would stick to the plan. Whatever is cheaper, manroof or amazon

**Comment by href** [comment]
Well I don't know for sure if manroof is cheaper because I will never understand the import tariffs which are probably set using /dev/urandom. I did contact them already and they are able to deliver in one or two days, so for peace of mind I'll probably go with them. Support local businesses and so on..

**Comment by href** [comment]
Date and location are a good idea!

![screenshot of sketch 20-01-16 17-49-36](https://cloud.githubusercontent.com/assets/273163/12456000/39c5868a-bf9e-11e5-8923-055c46dcc41d.png)

**Comment by dbrgn** [comment]
Love it!

Import tariffs are actually easy. There are none, for 99.9% of the products.

What you pay is VAT. The VAT is 8%. If the VAT is more than 5 CHF, the customs may collect it (and add a processing fee).
- 62 CHF \* 0.08 == 4.96 CHF
- 63 CHF \* 0.08 == 5.04 CHF

So if the value (including shipping) of the package is below 62.50 CHF, you will **not** pay any fees :)

As there's free shipping you could split it into 2 separate orders. I'd say we go for those on Amazon and do 2 orders. If @href orders tonight, they should arrive in time.

I don't know about laminating. But that would be lots of work. And we're less flexible.

I actually like the handwriting because people can get creative :)

**Comment by href** [comment]
I swear I got taxed for 30 bucks orders before. But different topic ;-)

Alright, I shall order from Amazon tomorrow.

**Comment by The-Compiler** [comment]
My CHF 0.02: It'd be nice if the printing was double-sided (e.g. by folding it in the middle), to motivate people to write their names on both sides. It's quite frustrating to see flipped badges with a white paper instead of a name :wink:

**Comment by jakeret** [comment]
I thought about this as well. At least the logo should be on both sides

**Comment by href** [comment]
The Amazon order is on the way to @dbrgn. It should arrive by February 1st at the latest. My company ordered three items in one package, so there might be taxes ;)

I will finish the design by next week and turn it into a PDF. I can fit two pages on one A4 page, so the resulting cut out would have to be folded along the right edge:

```
┌─────────────────────────────────────────────────────────────┐
│                                                             │
│                                                             │
│         ┌────────────────────┬────────────────────┐         │
│         │                    │                    │         │
│         │        Logo        │        Logo        │         │
│         │                    │                    │         │
│         ├────────────────────┼────────────────────┤         │
│         │                                         │         │
│         │                    │                    │         │
│         │                                         │         │
│         │                    │                    │         │
│         │                                         │         │
│         │                    │                    │         │
│         │                                         │         │
│         └────────────────────▼────────────────────┘         │
│                              > (fold)                       │
│                                                             │
└─────────────────────────────────────────────────────────────┘
```

**Comment by dbrgn** [comment]
> My company ordered three items in one package, so there might be taxes ;)

That's going to be 25-30 CHF :(

> I can fit two pages on one A4 page, so the resulting cut out would have to be folded along the right edge:

Cool. Maybe we can put more information about the conference itself on the back? e.g. the website.

**Comment by href** [comment]
> That's going to be 25-30 CHF :(

My boss's time was more valuable ;) You can give me the bill if your package is tagged by customs.

> Cool. Maybe we can put more information about the conference itself on the back? e.g. the website.

Depends, I thought the point of two sides was to have people write their names on both sides?

**Comment by dbrgn** [comment]
> My boss's time was more valuable ;) You can give me the bill if your package is tagged by customs.

Alright :)

> Depends, I thought the point of two sides was to have people write their names on both sides?

Yeah, maybe you can replace the date/location with the website on the backside and keep the rest identical :)

**Comment by href** [comment]
> Yeah, maybe you can replace the date/location with the website on the backside and keep the rest identical.

Good idea!

**Comment by href** [comment]
Release candidate: 
[badge.pdf](https://github.com/SwissPy/swisspycon/files/103098/badge.pdf)

This fits DIN A6 exactly, which the sleeves advertise to have. Since it might be a bit off I would wait with producing those until we get the sleeves - then do a test run.

**Comment by jakeret** [comment]
Isn't it possible to have two on one A4 page? This would minimize the number of cuts we might have to do

**Comment by href** [comment]
Theoretically yes, A4 is double the size of A6 after all. But many consumer grade printers won't print all the way to the edge. So usually you keep some white space around.

If we can print the badge on a printer that prints all the way to the border I can do a PDF that includes two badges. The only thing left to do would be the folding. @dbrgn Does HSR have a printer that can print A4s all the way to the border?

**Comment by jakeret** [comment]
I'm aware of that. We could get around this problem by having a small border around the card. Then we could even print doublesided, which would completely circumvent the folding part

**Comment by href** [comment]
This is a good idea, even though it utterly _destroys_ my design ;)
I'll have another example ready with double sided printing and white borders. I do hope we can print all the way to the border though.

**Comment by href** [comment]
[badge-two-sided.pdf](https://github.com/SwissPy/swisspycon/files/103403/badge-two-sided.pdf)

**Comment by dbrgn** [comment]
Double sided would work but then we need to make sure that the pages are exactly aligned. Should work though. But then we need also to make sure to use thicker paper.

The HSR printers are something like this https://www.usa.canon.com/internet/portal/us/home/support/details/copiers-mfps-fax-machines/support-black-white-multifunction/imagerunner-advance-8085 (not with all modules though). @sfkeller, can the printers print borderless?

**Comment by href** [comment]
> Double sided would work but then we need to make sure that the pages are exactly aligned. Should work though.

Works on my printer ;). If we use my two-sided example with the white borders we are also quite flexible with the cutting. So if the sides are not perfectly aligned or the card is a bit too big we can adjust.

**Comment by dbrgn** [comment]
DHL tried to deliver the sleeves today, wasn't at home. I'll get them next Monday or so.

VAT bill was 38.50 CHF.

**Comment by href** [comment]
Ok, you can hand me the bill as is next Wednesday. I need some kind of paper trail.

**Comment by href** [comment]
Where are we on the printing? I'm guessing you are doing the printing @dbrgn. Do you need help with it?

**Comment by dbrgn** [comment]
I'll get the sleeves today, then I'll test whether A6 fits. If it does, I can print it on Tuesday night.

@sfkeller can I print these at HSR again? My laser printer has mediocre color quality...

**Comment by dbrgn** [comment]
Actually the delivery didn't contain the sleeves, it was the t-shirt delivery... Let's hope they arrive in time.

**Comment by dbrgn** [comment]
They did not :) Hole puncher should have worked out.

@href, do you want to complain to the Amazon seller? Maybe there are even return possibilities?

**Comment by href** [comment]
I might. Still let me know when they arrive and if/how much you have to pay for import taxes.
